CONTRIBUTING.md

```diff
@@ -39,9 +39,37 @@ Do all your development or fixing work here.
 
 After all your development/fixing work is done, do not forget to add `Unit Test` and `Regression Test` around that. It will be nice if you can add an example of the new feature you have added.
 
+#### * Check your work after running all Unit and Regression Tests
+
+You should run all the unit tests by hitting the following command
+
+```
+mvn clean install
+```
+
+To run regression tests, you need to run a OpenShift/Kubernetes Cluster and after that
+
+Move to Regression Test directory
+
+```
+cd kubernetes-itests
```

Comment:
As we are mentioning module in profile, I think user can run `mvn clean verify -Popenshift-itests` from root directory